src/VisualStudio/Core/Def/Packaging/PackageSearchService.Update.cs

```diff
@@ -578,21 +605,52 @@ private AddReferenceDatabase CreateDatabaseFromBytes(byte[] bytes)
                 }
             }
 
-            private byte[] ParseDatabaseElement(XElement element)
+            private bool TryParseDatabaseElement(XElement element, out byte[] bytes)
             {
                 _service.LogInfo("Parsing database element");
                 var contentsAttribute = element.Attribute(ContentAttributeName);
                 if (contentsAttribute == null)
                 {
-                    throw new FormatException($"Database element invalid. Missing '{ContentAttributeName}' attribute");
+                    _service._reportAndSwallowException(
+                        new FormatException($"Database element invalid. Missing '{ContentAttributeName}' attribute"));
+
+                    bytes = null;
+                    return false;
                 }
 
+                var contentBytes = ConvertContentAttribute(contentsAttribute);
+
+                var checksumAttribute = element.Attribute(ChecksumAttributeName);
+                if (checksumAttribute != null)
+                {
+                    var expectedChecksum64 = checksumAttribute.Value;
+                    var expectedChecksum = Convert.FromBase64String(expectedChecksum64);
+                    using (var md5 = MD5.Create())
```

Comment:
Don't use MD5.  Even though we're not using it for cryptography, API scans will flag this for review time after time.